homeassistant/components/matter/binary_sensor.py

```diff
@@ -78,7 +77,6 @@ def _update_from_device(self) -> None:
         entity_description=MatterBinarySensorEntityDescription(
             key="ContactSensor",
             device_class=BinarySensorDeviceClass.DOOR,
-            name="Contact",
```

Comment:
This will now become a "Door" instead of "Contact", is that what was aimed for?

Reply:
Yeah, as these sensors are always door/window sensors